docs/miscellaneous.rst

```diff
@@ -361,7 +361,7 @@ Global Variables
 - ``suicide(address recipient)``: a deprecated alias to ``selfdestruct``
 - ``<address>.balance`` (``uint256``): balance of the :ref:`address` in Wei
 - ``<address>.send(uint256 amount) returns (bool)``: send given amount of Wei to :ref:`address`, returns ``false`` on failure
```

Comment:
I don't think there is a reason to remove `transfer` but not `send`.

Reply:
OK - that was an item on the "TODO"-list - a reason may be to consider ``send`` a low-level thing and make the user solely responsible for its use - but I agree, it's probably better to move it to ``address payable`` as well. I'll do that!